### DISCUSSION TITLE
New user here, is this safe to use with yostar’s client?

### DISCUSSION THREAD

**Discussion by neaaar** [MaaAssistantArknights/MaaAssistantArknights#3860] [Q&A]
Mainly asking cause of this point in the English AK TOS:

(xiii)
using cheats, exploits, automation
software, bots, hacks, mods, or any unauthorized third-party software that modifies or interferes with the Service or any User's game experience;

Would the assistant fall under the automation software category?

Thanks in advance

**Comment by ABA2396** [comment]
The Chinese client has never managed the script, and all the blocked numbers are changed data. I do not know how the yostart agent will be, but at least until now, the external client has not heard that the script has been blocked.

**Reply by ABA2396** [reply]
In contrast, the color script is relatively safe. After all, it recognizes the image and clicks on the image. The location of the click is also random within the range. At least less detectable than scripts recorded by users using the emulator's built-in macros.
